source/system/Memory.ooc

```diff
@@ -50,3 +52,6 @@ GlobalCleanup: class {
 		}
 	}
 }
+
+// Must be here because Mutex.ooc is loaded before Memory.ooc
```

Comment:
Oh yeah, never assume anything about the order of initialization of static data. Maybe it was not that good idea to have it registered in every module, maybe we should collect them all in `GlobalCleanup` ?